README.md

```diff
@@ -7,14 +7,24 @@ This repository contains files to test MultiQC with. These are used in the
 automated [Travis CI build tests](https://travis-ci.org/ewels/MultiQC)
 and can be used whilst developing for MultiQC.
 
+## Contributing
+
 To add new files to this repository, please follow the following procedure:
 
-1. Fork the repository to your own account
-2. Clone the repository locally
-3. Copy your log files to `MultiQC_TestData/data/modules/<module_name>`
-4. Commit and push your changes (`git add .` / `git commit -m "<message describing change>"` / `git push`)
-5. Create a Pull Request to bring your changes back into this repository.
-6. Delete your fork once the PR is accepted _(optional)_
+After forking the repository to your own GitHub account:
+
+```
```

Comment:
Minor hint for the future - you can add the language name after the three backticks with GitHub markdown to get syntax highlighting.

eg: 
<img src="https://user-images.githubusercontent.com/465550/43380141-441bf8a4-93d0-11e8-9551-e3c57fa455ec.png" width=70>

gives:

```bash
git clone https://github.com/<your github username>/MultiQC_TestData.git
cd MultiQC_TestData
cp /your/files/. ./data/modules/<module_name> 
git add .
git commit -m "<message describing change>"
git push
```

(not super exciting here, but very useful for code blocks)